test/CMakeLists.txt

```diff
@@ -10,21 +10,39 @@
 # prepare derivative works, distribute copies to the public, perform
 # publicly and display publicly, and to permit others to do so.
 
-add_library(catch2_define catch2_define.cpp)
+if(NOT Catch2_FOUND)
```

Comment:
:+1:

Is Catch2 currently a submodule? If so, we should remove it.

Reply:
It's not, as far as I could tell. In fact it looks like every configuration would download it.